physics/GWD/unified_ugwp.meta

```diff
@@ -1221,6 +1252,21 @@
   dimensions = ()
   type = logical
   intent = in
+[do_gwd_opt_psl]
+  standard_name = do_gsl_drag_suite_with_psl_gwd_option
+  long_name = flag to activate PSL drag suite - mesoscale GWD and blocking
+  units = flag 
+  dimensions = () 
+  type = logical
+  intent = in 
+[psl_gwd_dx_factor]
+  standard_name = effective_grid_spacing_of_psl_gwd_suite
+  long_name = multiplication of grid spacing
+  units = count
```

Comment:
According to https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst, the units should be `1` not count. Please make the change here and in fv3atm.

Reply:
changed